shared/src/main/scala/amf/client/remod/amfcore/plugins/render/AMFRenderPlugin.scala

```diff
@@ -0,0 +1,32 @@
+package amf.client.remod.amfcore.plugins.render
+
+import amf.client.plugins.AMFDocumentPlugin
+import amf.client.remod.amfcore.plugins.{AMFPlugin, PluginPriority}
+import amf.core.emitter.{RenderOptions, ShapeRenderOptions}
+import amf.core.model.document.BaseUnit
+import org.yaml.builder.DocBuilder
+
+private[amf] trait AMFRenderPlugin extends AMFPlugin[RenderInfo] {
+  def emit[T](unit: BaseUnit,
+              builder: DocBuilder[T],
+              renderOptions: RenderOptions = RenderOptions(),
+              shapeRenderOptions: ShapeRenderOptions = ShapeRenderOptions()): Boolean
+}
+
+
+private[amf] case class AMFRenderPluginAdapter(plugin: AMFDocumentPlugin) extends AMFRenderPlugin {
+
+  override def emit[T](unit: BaseUnit, builder: DocBuilder[T], renderOptions: RenderOptions, shapeRenderOptions: ShapeRenderOptions): Boolean =
```

Comment:
Isn't it weird that the `emit` method returns a boolean? I know it's for backwards compatibility but still. Weird.